docs/types/interfaces.md

```diff
@@ -198,16 +202,75 @@ class Company(Customer):
 
 When a field’s return type is an interface, GraphQL needs to know what specific
 object type to use for the return value. In the example above, each customer
-must be categorized as an `Individual` or `Company`. To do this you need to
-always return an instance of an object type from your resolver:
+must be categorized as an `Individual` or `Company`.
+
+### Returning a member instance
+
+To do this, you could return an instance of a member from your resolver:
 
 ```python
 import strawberry
 
 
+@strawberry.interface
+class Customer:
+    name: str
+
+
+@strawberry.type
+class Individual(Customer):
+    ...
+
+
+@strawberry.type
+class Company(Customer):
+    ...
+
+
 @strawberry.type
 class Query:
     @strawberry.field
     def best_customer(self) -> Customer:
         return Individual(name="Patrick")
+
+
+schema = strawberry.Schema(query=Query, types=[Individual, Company])
+```
+
+### Returning an interface instance
+
+You could also return the interface directly by defining a `resolve_type`
+method on the interface.
+
+```python
+import strawberry
+
+
+@strawberry.interface
+class Customer:
+    name: str
+
+    @classmethod
+    async def resolve_type(cls, obj: Any, info: Info):
```

Comment:
This will be invalid until the change to omit the 4th positional arg goes live.
CC @patrick91

Reply:
I wonder how easy that would be 🤔 I'm not sure if I'll have today to check the code